integration/integration_test.go

```diff
@@ -775,16 +776,34 @@ func TestOpensnoop(t *testing.T) {
 	t.Parallel()
 
 	opensnoopCmd := &command{
-		name:           "StartOpensnoopGadget",
-		cmd:            fmt.Sprintf("$KUBECTL_GADGET trace open -n %s", ns),
-		expectedRegexp: fmt.Sprintf(`%s\s+test-pod\s+test-pod\s+\d+\s+whoami\s+3`, ns),
-		startAndStop:   true,
+		name:         "StartOpensnoopGadget",
+		cmd:          fmt.Sprintf("$KUBECTL_GADGET trace open -n %s -o json", ns),
+		startAndStop: true,
+		expectedOutputFn: func(output string) error {
+			expectedEntry := &openTypes.Event{
+				Event: buildBaseEvent(ns),
+				Comm:  "cat",
+				Fd:    3,
+				Ret:   3,
+				Err:   0,
+				Path:  "/dev/null",
+			}
+
+			normalize := func(e *openTypes.Event) {
+				e.Node = ""
+				e.MountNsID = 0
+				e.Pid = 0
+				e.UID = 0
+			}
+
+			return expectEntriesToMatch(output, normalize, expectedEntry)
```

Comment:
Why not `expectAllToMatch`?

Reply:
I wasn't sure all events would match (there could be other open() calls in the container). I tried and it seems to work fine, so let's use that instead.

Reply:
I tested it as well and I didn't see any other entry.

Reply:
My initial concern was right after all -> https://github.com/kinvolk/inspektor-gadget/pull/984. :D